concepts/observability-strategy/configurable-tracing/pocs/instrumentation/README.md

```diff
@@ -0,0 +1,136 @@
+# OpenTelemetry Collector Performance
+
+The measured numbers that generated using version 0.60.0 of the OpenTelemetry Collector, are primarily applicable to the OpenTelemetry Collector and measured only for traces. In the future, more configurations should be tested.
+
+It is important to know, that the performance of the OpenTelemetry Collector depends on a set of factors including:
+
+- CPU / Memory allocation
+- The type of sampling: Tail based or head based sampling
+- The receiving format: OpenTelemetry, Jaeger thrift or Zipkin
+- The size of spans
+
+Please note, with OpenTelemetry agent expected better performance with lower resource utilization, since the OpenTelemetry Agent does not support features such as batching or retries and no tail based sampling supported yet.
```

Comment:
I'm not sure I understood the meaning of this sentence. Please correct me if I misunderstood the meaning.


```suggestion
With OpenTelemetry agent, better performance with lower resource utilization is expected, because the OpenTelemetry agent does not support features such as batching, retries, or tail-based sampling.
```

Reply:
No, sentence has still correct meaning